src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/ButtonManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Android.Views;
+using AView = Android.Views.View;
+
+namespace Xamarin.Platform
+{
+	public static class ButtonManager
+	{
+		public static bool OnTouch(IButton? button, AView? v, MotionEvent? e)
```

Comment:
Do we need this class currently?

Should these be part of the ButtonExtensions?